content/en/security_platform/application_security/setup_and_configure.md

```diff
@@ -190,9 +190,25 @@ The data that you’re collecting with Datadog can contain sensitive information
 
 The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans. See [APM Data Security][1] for details that also apply to Application Security.
 
+## Exclusion filters
```

Comment:
@arnaudbreton Bom dia! 👋  Opened this PR and added exclusion filters / uninstall content. Exclusion filters are not billed, correct? And is there a limit on the number of filters you can set? I'll add those notes in the PR once confirmed. Feel free to add any suggestions to this area, and uninstallation as well.

Reply:
Indeed, requests (traces) matching an exclusion filter won't be charged. 

About the limit of filters, I don't think so but I'm asking to the team. [Slack thread](https://dd.slack.com/archives/C01UQV7HA3A/p1645012496120029)